src/TrafficSign/TrafficSign.types.ts

```diff
@@ -163,7 +95,8 @@ export type TrafficSignProps = {
     | "urbanAreaEnd"
     | "warningPedestrianCrossing"
     | "weightLimit"
-    | "widthLimit";
+    | "widthLimit"
+    | string;
```

Comment:
The `| string` at the end of your union type makes the type effectively just string, as it allows any string value. Is that what you want?

Reply:
Yes, this is the desired behavior. The component is still protected internally from hard erroring. The purpose of this is since the type is sometimes a name + value like speedLimit20, which is created dynamically from the incoming JSON by concatenating two strings. This just results in TS issues.